integration/integration_test.go

```diff
@@ -89,13 +89,16 @@ func TestMain(m *testing.M) {
 
 	initCommands := []*command{
 		deployInspektorGadget,
+		deploySPO,
```

Comment:
Cannot this be done only for `TestAuditSeccomp()`?
Nonetheless, if we do it here, we would be able to test feature of `seccomp-advisor`.

Reply:
> Cannot this be done only for TestAuditSeccomp()?

It's possible. I'm fine either way. There was discussion about it in https://github.com/kinvolk/inspektor-gadget/pull/383#discussion_r826181110

> Nonetheless, if we do it here, we would be able to test feature of seccomp-advisor.

Yes :) In another PR?

Reply:
After some thoughts, I think it is better to keep SPO deploying into `TestMain()`, we can say we are writing `future-proof` code since we will use it to test `seccomp-advisor`.

> Yes :) In another PR?

Of course! This one is cool, so I will not ask you to add more code.